src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/CallWebApi.razor

```diff
@@ -0,0 +1,28 @@
+﻿@page "/callwebapi"
+
+@using BlazorServerWeb_CSharp
+
+@inject IDownstreamWebApi downstreamAPI
+
+<h1>Call an API</h1>
+
+<p>This component demonstrates fetching data from a Web API.</p>
+
+@if (apiResult == null)
+{
+    <p><em>Loading...</em></p>
+}
+else
+{
+    <h2>API Result</h2>
+    @apiResult
+}
+
+@code {
+    private string apiResult;
+
+    protected override async Task OnInitializedAsync()
+    {
+        apiResult = await downstreamAPI.CallWebApiAsync();
```

Comment:
We are working on making this work when there is a session cookie. Expect change in this particular file.